packages/cli/src/config/schema.ts

```diff
@@ -1012,6 +1016,53 @@ export const schema = {
 		},
 	},
 
+	eventBus: {
+		enabled: {
+			doc: 'Whether event logging is enabled.',
+			format: Boolean,
+			default: true,
+			env: 'N8N_EVENTBUS_ENABLED',
+		},
+		checkUnsentInterval: {
+			doc: 'How often (in ms) to check for unsent event messages. Can in rare cases cause a message to be sent twice. 0=disabled',
+			format: Number,
+			default: 0,
+			env: 'N8N_EVENTBUS_CHECKUNSENTINTERVAL',
+		},
+		destinations: {
+			loadAtStart: {
+				doc: 'Whether destinations are loaded from Db and initialized at startup.',
+				format: Boolean,
+				default: true,
+				env: 'N8N_EVENTBUS_DESTINATIONS_STARTUP',
+			},
+		},
+		logWriter: {
```

Comment:
Do you think it makes sense to merge those with the other logging settings?

Reply:
it's very much hard-linked with the eventbus, since it is what's launching the process with the log writer (unlike the loggerproxy, which I think runs through Watson). Since both are configuring totally different objects and logs, I'd keep them apart.